src/fides/api/ops/api/v1/scope_registry.py

```diff
@@ -6,161 +6,225 @@
-    CONNECTION_CREATE_OR_UPDATE,
-    CONNECTION_DELETE,
-    CONNECTION_AUTHORIZE,
-    SAAS_CONNECTION_INSTANTIATE,
-    CONSENT_READ,
-    CONNECTION_TYPE_READ,
-    DATABASE_RESET,
-    DATAMAP_READ,
-    DATASET_CREATE_OR_UPDATE,
-    DATASET_DELETE,
-    DATASET_READ,
-    ENCRYPTION_EXEC,
-    GENERATE_EXEC,
-    POLICY_CREATE_OR_UPDATE,
-    POLICY_READ,
-    POLICY_DELETE,
-    PRIVACY_REQUEST_CREATE,
-    PRIVACY_REQUEST_REVIEW,
-    PRIVACY_REQUEST_READ,
-    PRIVACY_REQUEST_DELETE,
-    PRIVACY_REQUEST_CALLBACK_RESUME,
-    PRIVACY_REQUEST_NOTIFICATIONS_CREATE_OR_UPDATE,
-    PRIVACY_REQUEST_NOTIFICATIONS_READ,
-    PRIVACY_REQUEST_TRANSFER,
-    PRIVACY_REQUEST_UPLOAD_DATA,
-    PRIVACY_REQUEST_VIEW_DATA,
-    RULE_CREATE_OR_UPDATE,
-    RULE_READ,
-    RULE_DELETE,
-    SCOPE_READ,
-    STORAGE_CREATE_OR_UPDATE,
-    STORAGE_DELETE,
-    STORAGE_READ,
-    MESSAGING_CREATE_OR_UPDATE,
-    MESSAGING_DELETE,
-    MESSAGING_READ,
-    WEBHOOK_CREATE_OR_UPDATE,
-    WEBHOOK_READ,
-    WEBHOOK_DELETE,
-    SAAS_CONFIG_CREATE_OR_UPDATE,
-    SAAS_CONFIG_READ,
-    SAAS_CONFIG_DELETE,
-    USER_CREATE,
-    USER_UPDATE,
-    USER_READ,
-    USER_PASSWORD_RESET,
-    USER_DELETE,
-    USER_PERMISSION_CREATE,
-    USER_PERMISSION_UPDATE,
-    USER_PERMISSION_READ,
-    VALIDATE_EXEC,
-]
+PRIVACY_REQUEST_NOTIFICATIONS_READ = f"{PRIVACY_REQUEST_NOTIFICATIONS}:{READ}"
+PRIVACY_REQUEST_READ = f"{PRIVACY_REQUEST}:{READ}"
+PRIVACY_REQUEST_REVIEW = f"{PRIVACY_REQUEST}:{REVIEW}"
+PRIVACY_REQUEST_TRANSFER = f"{PRIVACY_REQUEST}:{TRANSFER}"
+PRIVACY_REQUEST_UPLOAD_DATA = f"{PRIVACY_REQUEST}:{UPLOAD_DATA}"
+PRIVACY_REQUEST_VIEW_DATA = f"{PRIVACY_REQUEST}:{VIEW_DATA}"
+
+RULE_CREATE_OR_UPDATE = f"{RULE}:{CREATE_OR_UPDATE}"
+RULE_DELETE = f"{RULE}:{DELETE}"
+RULE_READ = f"{RULE}:{READ}"
+
+SAAS_CONFIG_CREATE_OR_UPDATE = f"{SAAS_CONFIG}:{CREATE_OR_UPDATE}"
+SAAS_CONFIG_DELETE = f"{SAAS_CONFIG}:{DELETE}"
+SAAS_CONFIG_READ = f"{SAAS_CONFIG}:{READ}"
+
+SAAS_CONNECTION_INSTANTIATE = f"{CONNECTION}:{INSTANTIATE}"
+
+SCOPE_READ = f"{SCOPE}:{READ}"
+
+STORAGE_CREATE_OR_UPDATE = f"{STORAGE}:{CREATE_OR_UPDATE}"
+STORAGE_DELETE = f"{STORAGE}:{DELETE}"
+STORAGE_READ = f"{STORAGE}:{READ}"
+
+SYSTEM_CREATE = f"{SYSTEM}:{CREATE}"
+SYSTEM_UPDATE = f"{SYSTEM}:{UPDATE}"
+SYSTEM_DELETE = f"{SYSTEM}:{DELETE}"
+
+TAXONOMY_CREATE = f"{TAXONOMY}:{CREATE}"
+TAXONOMY_UPDATE = f"{TAXONOMY}:{UPDATE}"
+TAXONOMY_DELETE = f"{TAXONOMY}:{DELETE}"
+
+USER_CREATE = f"{USER}:{CREATE}"
+USER_DELETE = f"{USER}:{DELETE}"
+USER_READ = f"{USER}:{READ}"
+USER_UPDATE = f"{USER}:{UPDATE}"
+USER_PASSWORD_RESET = f"{USER}:{PASSWORD_RESET}"
+
+USER_PERMISSION_CREATE = f"{USER_PERMISSION}:{CREATE}"
+USER_PERMISSION_UPDATE = f"{USER_PERMISSION}:{UPDATE}"
+USER_PERMISSION_READ = f"{USER_PERMISSION}:{READ}"
+
+VALIDATE_EXEC = f"{VALIDATE}:{EXEC}"
+
+WEBHOOK_CREATE_OR_UPDATE = f"{WEBHOOK}:{CREATE_OR_UPDATE}"
+WEBHOOK_DELETE = f"{WEBHOOK}:{DELETE}"
+WEBHOOK_READ = f"{WEBHOOK}:{READ}"
+
+SCOPE_DOCS = {
```

Comment:
Should we just drop the `SCOPE_DOCS` and just directly create the `SCOPE_REGISTRY` list in the `ops` style. I did a search and it doesn't look like we actually use `SCOPE_DOCS` anywhere so I don't see any benefit to it.

Reply:
That's funny, I compared them both yesterday and thought "oh that looks helpful". Granted the descriptions could use improvement in places.

Reply:
My thoughts were: we take a small performance hit to build and unpack the dictionary which seemed not worth it if we aren't using the dictionary, and these things seem to always get out of date and end up with incorrect descriptions. That said, I'm not strongly opposed to it either.